ssn/index.html

```diff
@@ -952,23 +975,54 @@ <h5><a href="#SOSAphenomenonTime">sosa:phenomenonTime</a></h5>
           <p><strong>an OWL Object Property</strong></p>
 
           <em property="rdfs:label">phenomenon time</em> -
-          <span property="rdfs:comment skos:definition">The time that the <a href="#SOSAResult"><code>Result</code></a>
-            of an <a href="#SOSAObservation"><code>Observation</code></a>, <a
-              href="#SOSAActuation"><code>Actuation</code></a>, or <a href="#SOSASampling"><code>Sampling</code></a>
-            applies to the <a href="#SOSAFeatureOfInterest"><code>FeatureOfInterest</code></a>. Not necessarily the same
-            as the <a href="#SOSAresultTime"><code>resultTime</code></a>. May be an interval or an instant, or some
-            other compound temporal entity [[!owl-time]].</span> <br>
+          <span property="skos:definition">The time that the result of an <a
+              href="#SOSAActuation"><code>Actuation</code></a>, <a href="#SOSAObservation"><code>Observation</code></a>,
+            or <a href="#SOSASampling"><code>Sampling</code></a>
+            applies to the <a href="#SOSAFeatureOfInterest"><code>FeatureOfInterest</code></a>. It may be an interval or
```

Comment:
If my proposal above is accepted, then the Actuation part needs to be adjusted accordingly.